packages/documentation/src/stories/components/header/renderers/custom-controls.ts

```diff
@@ -19,7 +19,7 @@ export function renderCustomControls(args: Args) {
           <post-icon aria-hidden="true" name="search"></post-icon>
         </a>
       </li>
-      <li>${args.isLoggedIn ? renderUserMenu() : loginButton}</li>
+      <li class="local-login">${args.isLoggedIn ? renderUserMenu() : loginButton}</li>
```

Comment:
The need for that class is a breaking change.
I'm fine with it, but you have to add a changeset 😉